update_ensembl_data.py

```diff
@@ -11,7 +11,19 @@
 
 import stdpopsim
 import maintenance
+import argparse
+import os
 
+# Parser that controls which species should be updated
+parser = argparse.ArgumentParser(description='Query ensembl API for chromosome lengths')
+parser.add_argument('--list-species', dest='list_species', action='store_true',
```

Comment:
I think we can simplify this a bit - the ``--list`` isn't really needed since we can get the list of species from a bunch of other places. What about if we had the default of doing what it's currently doing (all species), so if we call
```
python update_ensembl_data.py
```
it'll run through all species and if we call 
```
python update_ensembl_data.py HomSap BosTau
```
it'll just do those two. Basically, it's just a case of putting one positional argument, rather than two mutually exclusive mandatory "options".

Reply:
And the `--list` bit is just so that someone doesn't have to go hunting for a way to list all the registered species. I'm particularly thinking of my current case where a new developer is submitting a species but is not familiar with `stdpopsim` generally.

Reply:
Well, the update_all is harmless as it'll just over-write identical data:
```
$ python3 update_ensembl_data.py 
Using Ensembl release 101
Writing genome data for AraTha arabidopsis_thaliana
Writing genome data for CanFam canis_familiaris
Writing genome data for DroMel drosophila_melanogaster
... [Ctrl-c'd out of boredom]
$ git diff
diff --git a/stdpopsim/catalog/ensembl_info.py b/stdpopsim/catalog/ensembl_info.py
index 4800dda..1d96ff5 100644
--- a/stdpopsim/catalog/ensembl_info.py
+++ b/stdpopsim/catalog/ensembl_info.py
@@ -1,2 +1,2 @@
 # File autogenerated from Ensembl REST API. Do not edit.
-release = 100
+release = 101
```

So, we updated 3 species, but there's no diff in git (other than we seem to have updated to Ensembl release 101).

Having mandatory options (you must specify either --species or --update-all) gives me hives.

Fair enough having --list is handy, so let's keep that.

I have strong opinions about CLI structure I'm afraid @ndukler :wink:

Reply:
I'll change it. We can always catch a surprise unintentional update at the review stage.